src/pages/hegel/guides/science-of-logic-introduction.mdx

```diff
@@ -0,0 +1,184 @@
+
+Hegel notes that the manner one here views logic and its subject matter, thought and the world, subject and object, is in _phenomenal_ terms. But he calls it a prejudice to carry over this view into reason and logic. The argument is incoherent from the start since the phenomenal is allowed to make a metaphysical claim, but the former is exactly not the latter. The way one experiences things through the senses cannot be the standard for how everything is evaluated. Indeed, senses do not in the strict sense evaluate. This is not to say that experience is not real or important, but it is being misused if it is taken out of its scope. This miscarriage of logic is simply classed as an error and Hegel notes how it can be an impediment to philosophical thinking. 
+
+> ...when these prejudices are carried over to reason, as if in reason the same relation obtained, as if this relation had any truth in and for itself, then they are errors, and the refutation of them in every part of the spiritual and natural universe is what philosophy is; or rather,since they block the entrance to philosophy, they are the errors that must be removed before one can enter it (Hegel, 2010, 25/21.29).
+
+Curiously, the ancient thinkers, Hegel continues, had a higher conception of thinking since it was reasoned that only things known in thought, as opposed to known immediately, was understood to be the truth of the matter. It is, then, not what is _first_ experienced that is considered true, but what is _thought_ to be so. For example, in the distance an object may appear as a shipwreck but coming closer it becomes apparent that the object is actually a peculiar hut. Did the object itself morph between the two states? No, one reasons that perception gave a certain impression at one point and a different form another, but that the object had been what it was in both respects. One simply comes to _realize_ what the thing really was. Here thought is not alien to the world but is immanent to it, "the _things_ and the _thinking_ of them agree in and for themselves (also our language expresses a kinship between them)" (Hegel, 2010, 25/21.29). 
+
+### The Conflict of Determinations
+
+Hegel complains that the understanding has taken hold philosophy, and that, what is worse, ordinary common sense is granted dominion over the evaluating the truth about all things. Ordinary common sense being rooted in sensuous reality with the notion that thoughts are _only_ thoughts. Essentially, this view is uncritical and philosophically naive.
+
+However, Hegel detects a more serious obstacle in this thinking when it is attempted to be philosophically applied. When, according to this naive position, thoughts are _only_ thoughts and that real knowledge is the intertwining of thoughtful form and worldly impressions, it can be easily missed that what are assumed to be conflicts of the world are really conflicts of ideas. In Hegel's more technical vocabulary, it is _the conflict of determinations_.
```

Comment:
"are really conflicts of ideas" - a reference to Kant's transcendental dialectic? Could be nice to cite that since I also mention it in my presuppositionless article.

Reply:
Wasn't really thinking of kant but you're right. Added